pkg/certificates/certificates.go

```diff
@@ -1,14 +1,16 @@
 package certificates
 
 import (
+	"time"
+
 	"k8s.io/client-go/util/certificate"
 
 	"kubevirt.io/kubevirt/pkg/certificates/triple"
 	"kubevirt.io/kubevirt/pkg/certificates/triple/cert"
 )
 
 func GenerateSelfSignedCert(certsDirectory string, name string, namespace string) (certificate.FileStore, error) {
-	caKeyPair, _ := triple.NewCA("kubevirt.io")
+	caKeyPair, _ := triple.NewCA("kubevirt.io", time.Hour*24*7)
```

Comment:
maybe make whatever duration we end up going wiht here a const

Reply:
It is just used in tests. I is random.